tests/testthat/test-testthat-wrappers.R

```diff
@@ -0,0 +1,15 @@
+
+testthat::test_that("testthat wrappers return the object", {
+  expect_equal(
+    expect_message_obj({message("abc");"foo"}, "a"),
```

Comment:
These tests are implicit, not sure we need them.

We use styler to format the code, the result would look like this:

```suggestion
    expect_message_obj(
      {
        message("abc")
        "foo"
      },
      "a"
    ),
```

In particular:

- Each statement gets a separate line
- Multiline blocks are always indented